golf2/src/main.rs

```diff
@@ -425,9 +425,9 @@ pub unsafe fn reset_handler() {
     #[allow(unused)]
     let flash_test = static_init!(
         flash_test::FlashTest<
-            h1b::hil::flash::FlashImpl<'static, VirtualMuxAlarm<'static, Timels>>>,
+            h1::hil::flash::FlashImpl<'static, VirtualMuxAlarm<'static, Timels>>>,
         flash_test::FlashTest::<
-            h1b::hil::flash::FlashImpl<'static,
+            h1::hil::flash::FlashImpl<'static,
                                        VirtualMuxAlarm<'static, Timels>>>::new(flash));
```

Comment:
Alignment off by 1 (lines 430-431)

Reply:
You have really good eyes. I'm impressed!